robotbase/src/main/java/frc/robot/commands/pivot/SetPivotVisionTargeting.java

```diff
@@ -2,17 +2,51 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.Robot;
+import frc.robot.util.RobotMath;
 
+// TODO: Make this part of a larger command that handles the speed too.
 public class SetPivotVisionTargeting extends Command {
```

Comment:
We'll probably want to rename this to just `PivotVisionTargeting` and delete the `StopVisionTargeting` command if it has been made.

Reply:
Sounds good.